Jenkinsfile

```diff
@@ -1,107 +1 @@
-// This Jenkinsfile's main purpose is to show a real-world-ish example
-// of what Pipeline config syntax actually looks like.
-pipeline {
-    // Make sure that the tools we need are installed and on the path.
-    tools {
-        maven "mvn"
-        jdk "jdk8"
-    }
-
-    agent none
-
-    // Set log rotation, timeout and timestamps in the console
-    options {
-        buildDiscarder(logRotator(numToKeepStr:'10'))
-        timestamps()
-        timeout(time: 120, unit: 'MINUTES')
-    }
-
-    // Make sure we have GIT_COMMITTER_NAME and GIT_COMMITTER_EMAIL set due to machine weirdness.
-    environment {
-        GIT_COMMITTER_NAME = "jenkins"
-        GIT_COMMITTER_EMAIL = "jenkins@jenkins.io"
-        NEWER_CORE_VERSION = "2.164.3"
-        TEST_TIMEOUT = "600"
-    }
-
-
-    stages {
-        // While there is only one stage here, you can specify as many stages as you like!
-        stage("build") {
-            parallel {
-                stage("linux") {
-                    agent {
-                        label "highmem"
-                    }
-                    steps {
-                        sh "mvn -B clean install -Dmaven.test.failure.ignore=true -Djenkins.test.timeout=${TEST_TIMEOUT}"
-                    }
-                    post {
-                        // No matter what the build status is, run this step. There are other conditions
-                        // available as well, such as "success", "failed", "unstable", and "changed".
-                        always {
-                            junit testResults: '*/target/surefire-reports/*.xml', keepLongStdio: true
-                        }
-                        success {
-                            archive "**/target/*.hpi"
-                            archive "**/target/site/jacoco/jacoco.xml"
```

Comment:
IIRC there is an actual JaCoCo publisher available if anyone is looking at the results.

Reply:
@jglick But is it available on ci.jenkins.io?

Reply:
Looks like it is waiting for https://github.com/jenkins-infra/pipeline-library/pull/21. Not that the previous version actually did anything useful with the results; as you can see [here](https://ci.jenkins.io/job/Plugins/job/pipeline-model-definition-plugin/job/master/389/) for example, the raw output is available for download but there is no human-readable summary, `UNSTABLE` status when coverage regresses, etc.

Anyway this is a cross-cutting concern—nothing specific to this plugin.